compatibility/bazel_tools/testing.bzl

```diff
@@ -369,6 +369,20 @@ excluded_test_tool_tests = [
             },
         ],
     },
+    {
+        # Tests got renamed AGAIN in
+        # https://github.com/digital-asset/daml/commit/60ffb79fb16b507d4143cfc991da342efea504a7
+        "start": "1.17.0-snapshot.20210831.7702.1.f058c2f1",
+        "platform_ranges": [
+            {
+                "start": "1.17.0-snapshot.20210831.7702.1.f058c2f1",
+                "exclusions": [
+                    "CommandDeduplicationIT:ParticipantCommandDeduplicationSimpleDeduplicationBasic",
+                    "CommandDeduplicationIT:ParticipantCommandDeduplicationSimpleDeduplicationCommandClient",
+                ],
+            },
+        ],
+    },
```

Comment:
This seems to get the build to pass, but not having an end here feels wrong.

Reply:
This doesn’t make sense to me. A test being renamed should only cause one issue: It breaks existing exclusions that reference the test. It doesn’t create any incompatibilities.

So it sounds like the tests actually changed in a breaking way. In that case you usually need an exclusion for new ledger API test tool (meaning only a bound on start) versions against old ledger versions (meaning only a bound on end) if it only broke in one direction and if it also broke in the other direction you also need to exclude the other way.

Here you are excluding new ledger API test tool versions against new ledger versions which seems off.